Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -1104,7 +1105,7 @@ public override void Parse(string lowercasetoken, STFReader stf)
                 case "engine(dynamicbrakesresistorcurrentlimit": DynamicBrakeMaxCurrentA = stf.ReadFloatBlock(STFReader.UNITS.Current, null); break;
                 case "engine(numwheels": MSTSLocoNumDrvWheels = stf.ReadFloatBlock(STFReader.UNITS.None, 4.0f); if (MSTSLocoNumDrvWheels < 1) STFException.TraceWarning(stf, "Engine:NumWheels is less than 1, parts of the simulation may not function correctly"); break;
                 case "engine(ortsnumberdriveaxles": LocoNumDrvAxles = stf.ReadIntBlock(null); if (LocoNumDrvAxles < 1) STFException.TraceWarning(stf, "Engine:ORTSNumberDriveAxles is less than 1, parts of the simulation may not function correctly"); break;
-                case "engine(antislip": AntiSlip = stf.ReadBoolBlock(false); break;
+                case "engine(antislip": AntiSlip = stf.ReadBoolBlock(true); break;
```

Comment:
Another comment on AntiSlip parsing, it turns out MSTS accepted AntiSlip() when placed in the Wagon( section (look at the Dash 9, for example). While strange, this should also be allowed for compatibility with MSTS. MSTS even implements AntiSlip() on some wagon files, which probably means it's intended to work as wheel slide protection as well? Since OR already has ORTSWheelBrakeSlideProtection, AntiSlip could be an alias for that?
```suggestion
                case "engine(antislip":
                case "wagon(antislip":
                    AntiSlip = stf.ReadBoolBlock(true);
                    break;
```
I did some searching and this inconsistency, as well as the fact that "AntiSlip()" should be interpreted as "AntiSlip(true)", was actually [noticed many years ago](https://www.elvastower.com/forums/index.php?/topic/28692-antislip-parameter-not-correctly-parsed/) but it seems neither Carlo nor Peter acted on that discussion and it went forgotten.

Reply:
I don't think there's an added value in trying to match MSTS anymore. Legacy MSTS content will probably have AntiSlip() just because people copied that from other .eng or .wag file, not because the wagon had slip protection. Changing it now to accept the Wagon(AntiSlip()) will change the behavior of OR itself, which is a breaking change. If nobody complains about OR not accepting that, let's keep it as is.